### DISCUSSION TITLE
Typo/error in Holt Winters Section 7.3 additive example.

### DISCUSSION THREAD

**Discussion by kwojcicki** [OTexts/Discussion#2] [Error report]
Section https://otexts.com/fpp2/holt-winters.html#example-international-tourist-visitor-nights-in-australia

States:

> Table 7.3: Applying Holt-Winters’ method with additive seasonality for forecasting international visitor nights in Australia. Notice that the additive seasonal component sums to approximately zero. The smoothing parameters and initial estimates for the components have been estimated by minimising RMSE ( α = 0.306 ,  β∗=0.0003 ,  γ=0.426  and RMSE =1.763 )

However, running the equivalent code (I used https://rdrr.io/snippets/, not a plug but I don't have R installed locally and cannot be bothered to install it). shows that the beta star value should be 0.0001. 

```R

library(fpp2)

options(digits = 10)

aust <- window(austourists,start=2005)
fit1 <- hw(aust,seasonal="additive")
fit1$model
```

Which spits out the following:

```
Holt-Winters' additive method 

Call:
 hw(y = aust, seasonal = "additive") 

  Smoothing parameters:
    alpha = 0.3063 
    beta  = 1e-04 
    gamma = 0.4263 

  Initial states:
    l = 32.2597 
    b = 0.7014 
    s = 1.3106 -1.6935 -9.3132 9.6962
```

**Comment by robjhyndman** [comment]
$\beta^{\*}$ is not the same thing as $\beta$. 

In [Section 7.5](https://otexts.com/fpp2/ets.html), we change the notation setting $\beta = \alpha\beta^{\*}$. So $\beta^{\*} = \beta/\alpha = 0.0003$, which is what is given in the section you quote.

**Reply by kwojcicki** [reply]
Ah. I did recall in https://otexts.com/fpp2/holt.html#holts-linear-trend-method it was written that 

> (We denote this as  β∗ instead of  β  for reasons that will be explained in Section [7.5](https://otexts.com/fpp2/ets.html#ets).)

But I didn't get that far yet. My apologies :)
